meta-lmp-bsp/recipes-bsp/u-boot/u-boot-ostree-scr-fit/apalis-imx6/boot.cmd

```diff
@@ -7,22 +7,9 @@ echo "Using ${fdt_file_r}"
 setenv bootlimit 3
 setenv devtype mmc
 setenv devnum 0
+setenv rootpart 2
 
 setenv fdt_addr 0x15800000
 setenv optee_ovl_addr 0x16000000
 
-setenv bootcmd_resetvars 'setenv kernel_image; setenv bootargs; setenv kernel_image2; setenv bootargs2'
-setenv bootcmd_otenv 'run bootcmd_resetvars; ext4load ${devtype} ${devnum}:2 ${loadaddr} /boot/loader/uEnv.txt; env import -t ${loadaddr} ${filesize}'
-setenv bootcmd_load_f 'ext4load ${devtype} ${devnum}:2 ${loadaddr} "/boot"${kernel_image}'
-setenv bootcmd_tee_ovy 'imxtract ${loadaddr}#conf@@FIT_NODE_SEPARATOR@@${fdt_file_r} fdt@@FIT_NODE_SEPARATOR@@${fdt_file_r} ${fdt_addr}; fdt addr ${fdt_addr}; fdt resize 0x1000; fdt apply ${optee_ovl_addr}'
-setenv bootcmd_run 'bootm ${loadaddr}#conf@@FIT_NODE_SEPARATOR@@${fdt_file_r} ${loadaddr}#conf@@FIT_NODE_SEPARATOR@@${fdt_file_r} ${fdt_addr}'
```

Comment:
The apalis diff ^^. Just need to check what gets set in u-boot, and that is really board specific (some set fdtfile, others fdt_file_r, fdt_file, and even saw cases where they were all set to the same value).

Reply:
checked and resolved